### DISCUSSION TITLE
欢迎来讨论

### DISCUSSION THREAD

**Discussion by Binaryify** [Binaryify/NeteaseCloudMusicApi#1051] [General]
为了更好的进行讨论，说明下本讨论区规则

-只讨论项目相关的问题

-不要闲聊

-不要发布广告，维护良好的社区环境，谢谢。

**Comment by thirteen-wang** [comment]
请问，我想要在我的毕业设计中调用这套API，这是被允许的吗

**Reply by Binaryify** [reply]
可以的，不拿去商业用途就行

**Reply by thirteen-wang** [reply]
好的，谢谢您！因为没有经验，所以怕交项目时被导师毙掉

**Comment by helen-hjy** [comment]
您好！
首先感谢所有贡献者……一直想找网易云API很久了…… 
因为我一直在国外用Spotify，想把上面的歌曲导到网易云里，好像文档里我没有看到收藏歌曲到歌单欸……是可以直接“对歌单添加搜索的歌曲”嘛…… 然后现在好像没法clone？没有permission欸

**Comment by 1415desu** [comment]
可以提两个建议吗。
1 可以提供一个选项让关闭键最小化到托盘而非关闭吗
2 将软件固定到任务栏时，通过任务栏第二次打开后显示 该快捷方式所指向的项目"YesPlayMusic.exe"已经更改或移动，因此该快捷方式无法正常工作。可以解决一下这个问题吗？
3 可以保留网易云音乐的听歌识曲功能吗？

**Comment by IMDin** [comment]
请问，手机登录接口要怎么得到cookie~
是目前只有邮箱登录可以拿到吗，还是说手机登录有别的方法来得到登录凭证
可以解答的话，万分感谢！

**Reply by Binaryify** [reply]
都一样的

**Reply by IMDin** [reply]
我用的是/login/cellphone接口，在返回的信息中并没有找到cookie，浏览器也没有自动保存
请问是我操作失误了吗，传入手机密码其他信息都有返回，code为200
谢谢！

**Reply by Binaryify** [reply]
![image](https://user-images.githubusercontent.com/12221718/146329680-c0e5ec52-2dff-4cf6-9a8a-e398554bae75.png)

**Reply by IMDin** [reply]
谢谢！排查到问题了，我是封装了一下把所有接口都拦截器拦截了一遍，直接返回data的信息了，对不起，打扰了
（＞人＜；）

**Comment by ZhangWeijie-Z** [comment]
我也是用这个接口的，但我调用之后，会返回cookie，然后浏览器自动保存了登录信息，http://49.233.14.172:2888/static/music/index.html#/，这是我的线上项目，你可以在登录那里看看我的请求，是否和你的一样。
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Binaryify/NeteaseCloudMusicApi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月16日(星期四) 下午3:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Binaryify/NeteaseCloudMusicApi] 欢迎来讨论 (Discussion #1051)





 
我用的是/login/cellphone接口，在返回的信息中并没有找到cookie，浏览器也没有自动保存
 请问是我操作失误了吗，传入手机密码其他信息都有返回，code为200
 谢谢！
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by IMDin** [reply]
太贴心了，谢谢！

**Comment by ZhangWeijie-Z** [comment]
哈哈，没事，一起学习！


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Binaryify/NeteaseCloudMusicApi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月16日(星期四) 下午3:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Binaryify/NeteaseCloudMusicApi] 欢迎来讨论 (Discussion #1051)





 
谢谢！排查到问题了，我是封装了一下把所有接口都拦截器拦截了一遍，直接返回data的信息了，对不起，打扰了
 （＞人＜；）
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by xiaomougui** [comment]
很多热门音乐的url获取不到，检查音乐返回false，无版权，可是在网易云官网可以听，且有版权，请问这正常吗，是不是我写错了，或者是哪里没考虑到导致的问题

**Reply by xiaomougui** [reply]
希望楼主能看到，谢谢

**Reply by Binaryify** [reply]
要提供相关信息，服务器，请求参数
